frontend/main.js

```diff
@@ -95,7 +95,8 @@ async function startApp () {
         ephemeral: {
           syncs: [],
           // TODO/REVIEW page can load with already loggedin. -> this.$store.state.loggedIn ? 'yes' : 'no'
-          finishedLogin: 'no'
+          finishedLogin: 'no',
+          isCorrupted: true // TODO #761
```

Comment:
Can't merge while this is set to true; although having it set to true has helped me find a bug! 😄

Reply:
Changed to false!